tests/test_mod.py

```diff
@@ -11,6 +11,23 @@
 from sentinelsat.sentinel import (InvalidChecksumError, SentinelAPI, SentinelAPIError, convert_timestamp, format_date,
                                   get_coordinates, md5_compare)
 
+_small_query = dict(
+        area='0 0,1 1,0 1,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 1, 2))
+
+_large_query = dict(
+        area='0 0,0 10,10 10,10 0,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 12, 31))
+
+_api_auth = dict(
```

Comment:
@j08lue `pytest` will report skipped/xfailed tests as well, so they won't really fail silently. Still it would be a good idea to point this out in a small section of the readme aimed at developers.